src/components/SkillCard.js

```diff
@@ -4,7 +4,7 @@ function SkillCard({ card }) {
     return (
         <div className="skill-card">
             <i className={card.icon}></i>
-            <h3 className="headline section-subtitle"> {card.title} </h3>
+            <h3 className="headline card-title"> {card.title} </h3>
```

Comment:
This makes sense... the skill cards are their own entities. I think it's best to separate them from general subtitles through the page.